### PR TITLE
Added influx helper reporting script to utilities directory

### PR DESCRIPTION
**Pull request by hagertnl**

### CODE REVIEW

**Comment by hagertnl** [comment]
Added the update_runs_from_slurm.py, since it is also based in the utilities directory. This script checks for running jobs in Influx and compares that against what SLURM knows. This resolves walltimed jobs, cancelled jobs, node failures, and other errors that cause InfluxDB to fall out of date. Requires a local harness_keys.py file.

**Comment by hagertnl** [comment]
Switching between dev and prod instances are not as easy with this update_runs_from_slurm.py, since there's no command-line flag to enable it. Since this file needs both the GET and POST URLs for the Influx instance, I don't have an option to toggle between them at the moment.

File: harness/utilities/add_comment_to_influx.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+import subprocess
+import urllib.parse
+from datetime import datetime
+import argparse
+
+
+
+# Initialize argparse ##########################################################
+parser = argparse.ArgumentParser(description="Updates harness run in InfluxDB with a comment")
+parser.add_argument('--jobid', '-j', nargs=1, action='store', required=True, help="Specifies the harness jobID to update jobs for.")
```
Comment (hagertnl) [inline harness/utilities/add_comment_to_influx.py]: Don't use jobid for anything that's not a job :) Use testid

Comment (verolero86) [inline harness/utilities/add_comment_to_influx.py]: Is this the harness uniqueID or the Slurm/LSF jobID?

File: harness/utilities/add_comment_to_influx.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    if not len(values) == 1:
+        print(f"Query returned {len(values)} results, expected 1. Exiting.")
+        sys.exit(1)
+    # Let's do the work of transforming this into a list of dicts
+    ret_data = {}
+    for c_index in range(0, len(col_names)):
+        ret_data[col_names[c_index]] = values[0][c_index]
+    should_add, reason = check_data(ret_data)
+    if not should_add:
+        jobid = data_tmp['test_id'] if 'test_id' in data_tmp else 'unknown'
```
Comment (verolero86) [inline harness/utilities/add_comment_to_influx.py]: here jobid appears to be the harness uniqueID

**Review by verolero86 — COMMENTED** [review]
Thanks @hagertnl ! Looks good to me -- but it appears some `jobid`s are still ambiguous.

**Review by verolero86 — APPROVED** [review]
LGTM! 👍

**Comment by verolero86** [comment]
We will need a new PR to merge to devel but this looks good to me! Thank you @hagertnl !

**Pull request merged**